### PR TITLE
Update links.

### PR DESCRIPTION
**Pull request by julianschacherpp**
Update all references of the old domain to be references of the new one.
(https://www.algorithm-archive.org/)

### CODE REVIEW

**Comment by leios** [comment]
Thanks!

File: chapters/introduction.md [hunk 1]
```diff
@@ -12,8 +12,10 @@ This text is a natural extension of that idea -- a community effort to learn as
 
 For this reason, I hesitate to call this a work a "textbook" and would prefer to think of it as a lab notebook of sorts. It will be an archive filled with cool ideas in computer science and physics that I hope will be interesting in their own right. My hope is that this book perfectly straddles the line between comprehensiveness and pleasure and will be an enjoyable read for anyone at any stage on their hobby programming journey.
 
-Keeping with the community theme, this book is freely available on [github](https://github.com/algorithm-archivists/algorithm-archive) and [github pages](https://algorithm-archivists.github.io). Please feel free to read / browse the content available there. If you have any questions / comments / concerns, please let me know. This book is a community project and will only work if there is a community that works on it!
```
Comment (julianschacherpp) [inline chapters/introduction.md]: About this one: Should the link still be called `github pages`?

**Comment by julianschacherpp** [comment]
I commented on that PR something. Don't know if it shows here.

**Comment by leios** [comment]
Oops, should have left the PR up for longer. I thought it was a quick fix and there wasn't a need for discussion. I think github pages is still fine in this case, but I suppose we should have changed it to just "algorithm-archive.org"

**Comment by julianschacherpp** [comment]
Yeah, that's probably better. You can just commit it yourself then.
Sorry, should've mentioned that comment in the original PR text.

**Pull request merged**